build/systemd_units/manageiq-ems_metrics_processor.target

```diff
@@ -0,0 +1,2 @@
+[Unit]
+PartOf=miq.target
```

Comment:
Where is this target file?  Also, can we use `manageiq.target`?

Reply:
I see it's been miq.target all along.  I'm fine with this PR being `miq.target` to avoid churn and/or upgrade issues, but wonder if we can change in the future.

Reply:
Now that we have a `manageiq-` prefix on these units I think `manageiq.target` makes more sense

Reply:
> to avoid churn and/or upgrade issues

We're already going to have to blow away the old files on upgrade to support the `manageiq-*` stuff in the rpm build so might as well change it